src/main/java/Hello.java

```diff
@@ -6,24 +6,60 @@
 public class Hello {
 
   // Set SDK_KEY to your LaunchDarkly SDK key.
-  static final String SDK_KEY = "";
+  static String SDK_KEY = "";
 
   // Set FEATURE_FLAG_KEY to the feature flag key you want to evaluate.
-  static final String FEATURE_FLAG_KEY = "my-boolean-flag";
+  static String FEATURE_FLAG_KEY = "sample-feature";
   
   private static void showMessage(String s) {
     System.out.println("*** " + s);
     System.out.println();
   }
 
+  private static void printLogo() {
+    showMessage("\n        ██       \n" +
+                "          ██     \n" +
+                "      ████████   \n" +
+                "         ███████ \n" +
+                "██ LAUNCHDARKLY █\n" +
+                "         ███████ \n" +
+                "      ████████   \n" +
+                "          ██     \n" +
+                "        ██       \n");
+  }
+
   public static void main(String... args) throws Exception {
+    // Set this environment variable to skip the loop process and evaluate the flag
+    // a single time.
+    boolean CIMode = false;
+    String ci = System.getenv("CI");
+    if(ci != null) {
+      CIMode = true;
+    }
```

Comment:
I think you could probably tighten this up a bit.
```suggestion
    boolean CIMode = System.getenv("CI") != null;
```